cookiecutter.json

```diff
@@ -20,5 +20,9 @@
     "use_compressor": "n",
     "js_task_runner": ["Gulp", "Grunt", "None"],
     "use_lets_encrypt": "n",
+    "use_gitlab_ci": "n",
+    "staging_branch": "",
+    "staging_domain_name": "beta.{{ cookiecutter.domain_name }}",
+    "production_branch": "",
```

Comment:
Adding 4 questions is a lot, especially for anyone unfamiliar with GitLab's system (most of our users). That's going to add a huge burden of maintenance to the core contributors.

I would much rather have a single question (`use_gitlab_ci`) in `cookiecutter.json`, then see more questions asked during a `post-generate` hook.